### ISSUE TITLE
DeployOrchestrationToGac missing from .targets 

### ISSUE THREAD

**Issue by NealWalters** [BTDF/DeploymentFramework#452]
This is one of those case where everything was working great, then one day it suddenly broke. 

When I deploy to another machine using the generated BTDF MSI, the first server runs fine, but the second server gets an error on DeployOrchestrationToGac missing in the .targets file.  I opened it, and sure enough it is missing but the others are there. DeploySchemasToGac, DeployPipelinesToGac, and DeployTransformsToGac all ran okay. 

My .btdfproj file has these settings: 
`    <IncludeSchemas>True</IncludeSchemas>
    <IncludePipelines>True</IncludePipelines>
    <IncludeOrchestrations>True</IncludeOrchestrations>
    <IncludeTransforms>True</IncludeTransforms>
    <IncludeComponents>False</IncludeComponents>
    <UsingMasterBindings>True</UsingMasterBindings>
    <RequireXmlPreprocessDirectives>False</RequireXmlPreprocessDirectives>
    <ApplyXmlEscape>True</ApplyXmlEscape>
    <ModifyNTFSPermissionsOnVDirPaths>False</ModifyNTFSPermissionsOnVDirPaths>
    <EnableAllReceiveLocationsOnDeploy>True</EnableAllReceiveLocationsOnDeploy>
    <SkipHostInstancesRestart>False</SkipHostInstancesRestart>
    <AutoTerminateInstances>False</AutoTerminateInstances>
    
    <IncludeVirtualDirectories>False</IncludeVirtualDirectories>
    <IncludeInstallUtilForComponents>True</IncludeInstallUtilForComponents>
    <IncludeSettingsSpreadsheetInMsi>True</IncludeSettingsSpreadsheetInMsi>
    <SkipIISReset>True</SkipIISReset>
    <StartApplicationOnDeploy>True</StartApplicationOnDeploy>
    <StartReferencedApplicationsOnDeploy>True</StartReferencedApplicationsOnDeploy>
    <UndeployIISArtifacts>False</UndeployIISArtifacts>
    <IncludeSSO>True</IncludeSSO>`

So how does the .targets file get built for the target system.  Is there a copy of it on my development machine that maybe got messed up?  Or some flag I might have messed up in the .btdfproj file? 

Thanks,
Neal Walters

**Comment by tfabraham** [comment]
Hi Neal,
BizTalkDeploymentFramework.targets is entirely static.  It's simply packaged into the MSI on the machine that builds the MSI.  When you're using an MSI to deploy on a server, the .targets file that's used is in the app installation folder (typically) under Program Files.  On a machine that has the BTDF MSI installed, the .targets file is under Program Files(x86)\MSBuild\DeploymentFrameworkForBizTalk (or similar).

Tom

**Comment by NealWalters** [comment]
Thanks Tom.  Just solved - kind of.  
"c:\Program Files (x86)\MSBuild\DeploymentFrameworkForBizTalk\5.0\BizTalkDeploymentFramework.targets" (on my developer VM). 

The file above was updated on 08/01/2018.  And it's like someone went into it and just remove the target called DeployOrchestrationToGac.  I can't imagine why I would have done that, unless I was in some hypno-trance.  

I just restored the file from a buddy's machine, and about to test on next deploy; very confident that's what the problem was. 

Neal
